src/app/kvm/components/RamResources/RamResources.test.tsx

```diff
@@ -93,16 +69,17 @@ describe("RamResources", () => {
       />
     );
 
-    expect(wrapper.find("DoughnutChart").prop("segments")).toStrictEqual([
-      { color: COLOURS.CAUTION, value: 1 },
-    ]);
+    // eslint-disable-next-line testing-library/no-container
+    expect(container.querySelector(".doughnut-chart__segment")).toHaveStyle(
```

Comment:
Same as what I said earlier - I'm guessing the element can't be queried by a role or by label text, ideally we should avoid direct container access